content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -9,44 +9,99 @@ weight: 210
 ---
 
 <!-- overview -->
-This page shows how to enable and configure encryption of secret data at rest.
+
+Some of the APIs in Kubernetes, such as {{< glossary_tooltip text="Secret" term_id="secret" >}},
+support at-rest encryption. This at-rest encryption is additional to any system-level
+encryption for the etcd cluster or for the filesystem(s) on hosts where you are running the
+kube-apiserver.
+
+This page shows how to enable and configure encryption of API data at rest.
+
+{{< note >}}
+This task covers encryption for data stored using the
+{{< glossary_tooltip text="Kubernetes API" term_id="kubernetes-api" >}}.
+
+If you want to encrypt data in filesystems that are mounted into containers, you instead need
+to either:
+
+- use a storage integration that provides encrypted
+{{< glossary_tooltip text="volumes" term_id="volume" >}}
+- encrypt the data within your own application
+{{< /note >}}
 
 ## {{% heading "prerequisites" %}}
 
-* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+* {{< include "task-tutorial-prereqs.md" >}}
 
-* etcd v3.0 or later is required
+* This task assumes that you are running the Kubernetes API server as a
+  {{< glossary_tooltip text="static pod" term_id="static-pod" >}} on each control
+  plane node.
+
+* Your cluster's control plane **must** use etcd v3.x (major version 3, any minor version)
 
 * To encrypt a custom resource, your cluster must be running Kubernetes v1.26 or newer.
 
 
 <!-- steps -->
 
-## Configuration and determining whether encryption at rest is already enabled
+## Determine whether encryption at rest is already enabled {#determining-whether-encryption-at-rest-is-already-enabled}
+
```

Comment:
Since this is a *task* page, I'd like clear steps to complete some tasks.
Should this section list the items that you need to check before creating an encryption-at-rest configuration?

Not sure what to do using this sentence:
By default, the `identity` provider used by the API server to protect confidential data ...

Reply:
This page is in the tasks section but it is, unfortunately, also serving as a conceptual explanation. It would be nice to separate these things out; however, I think that's best saved for a separate PR.